analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ToStringShouldNotReturnNull.vb

```diff
@@ -63,22 +63,30 @@ Namespace Compliant
             Return String.Empty
         End Function
     End Structure
-	
-	Class ToString
-	
-		Public Function SomeMethod() As String
+
+    Class ToString
+
+        Public Function SomeMethod() As String
             Return Nothing 'Compliant
         End Function
-	
-	End Class
-	
-	Class BinaryConditionalExpressionNotSupported
-	
-		Public Function SomeMethod() As String
+
+    End Class
+
+    Class BinaryConditionalExpressionNotSupported
+
+        Public Function SomeMethod() As String
             Return If(Nothing, Nothing) ' Not supported, doesn't make sense
         End Function
-	
-	End Class
+
+    End Class
+
+    Class ToStringSharedMethod
```

Comment:
There is still a missing UT that is not raising because the function is shared.
This example would not raise even if it would not be shared as it returns String.Empty instead of Nothing.